app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -50,6 +51,8 @@ class ManageIQ::Providers::Openstack::CloudManager < ManageIQ::Providers::CloudM
                     :ensure_cinder_managers,
                     :ensure_swift_managers
 
+  OrchestrationTemplateCfnDialogService = OrchestrationTemplateHotDialogService
```

Comment:
OpenStack support both Cfn and Hot format but the service dialog should be the same. So here I made `OrchestrationTemplateCfnDialogService` an alias of `OrchestrationTemplateHotDialogService`